gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages.properties

```diff
@@ -117,11 +117,6 @@ overview_footnote_legend1=Titles shown in grey are not released to students.
 overview_footnote_legend2=Class averages in parentheses () are not included in the course grade calculations.
 overview_footnote_legend3=These items must be graded from the tool listed and may not be graded from the gradebook.
 overview_from=from
-overview_lettergrade1=We noticed that you are using letter grades in the gradebook and want to make sure that you are aware of the 
-overview_lettergrade2=explicit approach that Oncourse uses when handling letter grades.
```

Comment:
Is this deleting without replacing?

Reply:
Yes, I did a search and these strings are never rendered in the UI so I guess they can be deleted.

The strings refer to some kind of technical support, I did some tests in the UI and they are never displayed.